### ISSUE TITLE
tokei-6.1.2 build failure on macOS 10.12

### ISSUE THREAD

**Issue by kballard** [NixOS/nixpkgs#33668]
## Issue description
I tried to install tokei v6.1.2 on macOS 10.12 and got a reproducible build failure.

### Steps to reproduce
1. `nix-env -i tokei`

### Output
```
installing ‘tokei-6.1.2’
these derivations will be built:
  /nix/store/z9pzyw4qhs1mfx6v7ryvx7crxh7kg6v0-tokei-6.1.2.drv
building path(s) ‘/nix/store/xmmw0fz3j762x1fnpd1hksh96lpx2wzk-tokei-6.1.2’
unpacking sources
unpacking source archive /nix/store/09127whsbhwvb74svj08z61xahnc558f-source
source root is source
unpacking source archive /nix/store/jl4r0nj2j5vy3mdyw3bj9b1s7mzjbs41-tokei-6.1.2-vendor
patching sources
configuring
building
   Compiling libc v0.2.30
   Compiling num-traits v0.1.40
   Compiling futures v0.1.15
   Compiling unicode-width v0.1.4
   Compiling crossbeam v0.2.10
   Compiling utf8-ranges v1.0.0
   Compiling void v1.0.2
   Compiling rayon-core v1.2.1
   Compiling quick-error v1.2.0
   Compiling ansi_term v0.9.0
   Compiling vec_map v0.8.0
   Compiling regex-syntax v0.4.1
   Compiling dtoa v0.4.2
   Compiling strsim v0.6.0
   Compiling either v1.1.0
   Compiling fnv v1.0.5
   Compiling bitflags v0.9.1
   Compiling lazy_static v0.2.8
   Compiling errln v0.1.0
   Compiling log v0.3.8
   Compiling unicode-segmentation v1.2.0
   Compiling pest v0.3.3
   Compiling itoa v0.3.3
   Compiling serde v1.0.11
   Compiling encoding_index_tests v0.1.4
   Compiling scopeguard v0.3.2
   Compiling same-file v0.1.3
   Compiling unreachable v1.0.0
   Compiling num_cpus v1.6.2
   Compiling rand v0.3.16
   Compiling term_size v0.3.0
   Compiling memchr v1.0.1
   Compiling atty v0.2.2
   Compiling encoding-index-singlebyte v1.20141219.5
   Compiling encoding-index-japanese v1.20141219.5
   Compiling encoding-index-simpchinese v1.20141219.5
   Compiling encoding-index-tradchinese v1.20141219.5
   Compiling encoding-index-korean v1.20141219.5
   Compiling coco v0.1.1
   Compiling thread_local v0.3.4
   Compiling walkdir v1.0.7
   Compiling textwrap v0.7.0
   Compiling aho-corasick v0.6.3
   Compiling encoding v0.2.33
   Compiling clap v2.26.0
   Compiling rayon v0.7.1
   Compiling serde_json v1.0.2
   Compiling regex v0.2.2
   Compiling env_logger v0.4.3
   Compiling handlebars v0.27.0
   Compiling globset v0.2.0
   Compiling ignore v0.2.2
   Compiling tokei v6.1.2 (file:///private/tmp/nix-build-tokei-6.1.2.drv-0/source)
error: failed to decompress bc of `std-20c1862ba6fa8d28.0.bytecode.deflate`

error: aborting due to previous error

error: Could not compile `tokei`.

To learn more, run the command again with --verbose.
builder for ‘/nix/store/z9pzyw4qhs1mfx6v7ryvx7crxh7kg6v0-tokei-6.1.2.drv’ failed with exit code 101
error: build of ‘/nix/store/z9pzyw4qhs1mfx6v7ryvx7crxh7kg6v0-tokei-6.1.2.drv’ failed
```

## Technical details
```
 - system: `"x86_64-darwin"`
 - host os: `Darwin 16.7.0, macOS 10.12.6`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 1.11.16`
 - channels(root): `"nixpkgs-18.03pre122433.8a7b2f889dd"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

**Comment by dywedir** [comment]
related https://github.com/ogham/exa/issues/335

**Comment by kballard** [comment]
Also fails to install on macOS 10.13.2 (with the same error).

**Comment by matthewbauer** [comment]
Fixed with tokei-7.0.0
